### PR TITLE
Add namespaces

### PR DESCRIPTION
**Pull request by juliendangers**
# WIP

I need namespace support because of a conflict with Log class.

I you prefer another namespace, just tell me.

And do I need to do anything else ? (sorry I'm not a PHP developer)

## TODO

 - [x] fix decryption issue
 - [ ] fix examples

### CODE REVIEW

File: src/smileys.class.php [hunk 1]
```diff
@@ -22,7 +24,7 @@ class Smileys
     private $listSmileys = [
         'SMILING_FACE_WITH_OPEN_MOUTH_AND_SMILING_EYES'                      => 0x1F604, // Smiley 1
         'SMILING_FACE_WITH_OPEN_MOUTH'                                       => 0x1F603, // Smiley 2
-        'GRINNING_FACE_WITH_SMILING_EYES'                                    => 0x1F600, // Smiley 3
```
Comment (juliendangers) [inline src/smileys.class.php]: duplicate of line 40

File: src/smileys.class.php [hunk 2]
```diff
@@ -481,7 +483,6 @@ class Smileys
         'BASKETBALL_AND_HOOP'                                                => 0x1F3C0, // Smiley 459
         'SOCCER_BALL'                                                        => 0x26BD,  // Smiley 460
         'BASEBALL'                                                           => 0x26BE,  // Smiley 461
-        'GREEN_BOOK'                                                         => 0x1F4D7, // Smiley 462
```
Comment (juliendangers) [inline src/smileys.class.php]: duplicate of line 450

File: src/whatsprot.class.php [hunk 3]
```diff
@@ -2897,7 +2886,7 @@ public function sendMessageRead($to, $id)
     {
         $listNode = null;
         $idNode = $id;
-        if (is_array($id) && count($id > 1)) {
+        if (is_array($id) && count($id) > 1) {
```
Comment (juliendangers) [inline src/whatsprot.class.php]: was-it an error ?

**Comment by juliendangers** [comment]
@mgp25 for libaxolotl-php namespacing, do I need to create a PR on mgp25/libaxolotl-php too ? I will have to generate protobuf files with namespace too

File: examples/Simple CLI client.php [hunk 4]
```diff
@@ -111,7 +115,7 @@ public function __construct($wp, $target)
         $this->target = $target;
     }
 
-    public function process(\ProtocolNode $node)
+    public function process(ProtocolNode $node)
```
Comment (balnarz) [inline examples/Simple CLI client.php]: [Installing the system.txt](https://github.com/mgp25/Chat-API/files/377870/Installing.the.system.txt)

**Comment by BurakDev** [comment]
@0xTryCatch Why you close this PR ?

**Comment by mgp25** [comment]
The code in github of Chat API is outdated, we have a newer one. Trying to make standards to this one is a waste of time.

Maybe it could be useful in the future but not right now

Regards

**Pull request closed without merging**